test/search.spec.js

```diff
@@ -36,7 +36,7 @@ describe('Search', () => {
     let query = (temp.getMonth() + 1) + '-' + temp.getDate();
     request.get('/api/documents?limit=5&date=2016-' + query).set('x-access-token', token).set('Accept', 'application/json').expect(200).end(
       (req, res) => {
-        let result = new Date('2016' + query);
+        let result = new Date('2016-' + query);
         result.setDate(result.getDate() + 1);
         expect(res.body).to.have.length.of.at.most(5);
         expect(new Date(res.body[0].createdAt)).to.be.below(result);
```

Comment:
### Critical

This test case should ONLY test that documents returned are not more than the specified limit. It's not clear what this line seeks to achieve. Simply modify this to check that all results returned match the EXACT date that was specified in the request